### PR TITLE
Issue #3884 add D3 Scaling.scaleLinear()

### LINKED ISSUE #3884: [Term Entry] D3 Scaling .scaleLinear()
### Reason/inspiration (optional)

We would like a new term entry on the `.scaleLinear()` method for [D3](https://www.codecademy.com/resources/docs/d3): Scaling. The entry should go in a new file under `docs/content/d3/concepts/scaling/terms/scaleLinear/scaleLinear.md`.

The entry should describe what the method does. Be sure to include the following sections:

- A "Syntax" section with a ` ```pseudo` code snippet.
- An "Example" section with a ` ```js `code snippet.

Please refer to the [entry templates](https://github.com/Codecademy/docs/blob/main/documentation/), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [ ] Issue is assigned

### PR DESCRIPTION
**Pull request by aciffone23**
### Description
Closes #3884 
* Added `Scaling.scaleLinear() term entry`, `docs/content/d3/concepts/scaling/terms/scaleLinear/scaleLinear.md`. 
* Was motivated to make a contribution from the Learn Git & GitHub coursework. 

### Type of Change

- Adding a new entry
- Updating the documentation

### Checklist

- [X] All writings are my own.
- [X] My entry follows the Codecademy Docs style guide.
- [X] My changes generate no new warnings.
- [X] I have performed a self-review of my own writing and code.
- [X] I have checked my entry and corrected any misspellings.
- [X] I have made corresponding changes to the documentation if needed.
- [X] I have confirmed my changes are not being pushed from my forked `main` branch.
- [X] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [X] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/d3/concepts/scaling/terms/scaleLinear/scaleLinear.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+Subjects:
+  - 'Data Visualization'
+  - 'Data Science'
+Tags:
+  - 'Methods'
+  - 'Data'
+  - 'Graphs'
+CatalogContent:
+  - 'learn-d3'
+  - 'paths/computer-science'
```
Comment (dakshdeepHERE) [inline content/d3/concepts/scaling/terms/scaleLinear/scaleLinear.md:13]:
```suggestion
  - 'paths/data-science'
```

File: content/d3/concepts/scaling/terms/scaleLinear/scaleLinear.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  .range([minOutput, maxOutput]);
+```
+
+- `.scaleLinear()`: The method that creates a linear scale for continious mapping.
+- `.domain()`: Requires the `minValue` and `maxValue` arguments to set the accepted range.
+- `.range()`: The second step in defining a scale, the arguments `minOutput` and `maxOutput` are used to specify the output range of the scale.
+
+## Example
+
+In the example given below the `scale` will map values within the specified domain to the corresponding range given. The `scale` is set to have a domain from `0` to `100` and range from `0` to `500`. When the values are mapped from the domain passed to the scale, the range produces the output accordingly.
```
Comment (dakshdeepHERE) [inline content/d3/concepts/scaling/terms/scaleLinear/scaleLinear.md:32]:
```suggestion
In the example given below the `scale` will map values within the specified domain to the corresponding range given. The `scale` is set to have a domain from `0` to `100` and range from `0` to `500`. When the values are mapped from the domain passed to the scale, the range produces the output accordingly:
```

**Review by dakshdeepHERE — CHANGES_REQUESTED** [review]
Few changes from my side for you @aciffone23

**Comment by aciffone23** [comment]
> Few changes from my side for you @aciffone23

Thank you, committed the changes.

**Review by dakshdeepHERE — APPROVED** [review]
Looks good for second review 🚀

**Review by avdhoottt — CHANGES_REQUESTED** [review]
@aciffone23 Just a minor change from my side.

**Review by avdhoottt — APPROVED** [review]
Ready to merge.

**Pull request merged**